packages/core/src/App/Components/Elements/Modals/RedirectNotice/redirect-notice-modal.jsx

```diff
@@ -65,7 +70,8 @@ const RedirectNoticeModal = ({ is_logged_in, is_eu }) => {
             >
                 {localize('You are being redirected to an external website.')}
             </Dialog>
-        )
+        ) : null,
+        document.getElementById('popup_root')
```

Comment:
Maybe we can declare `popup_root` as a variable first and then refer it here as `popup_id` or something

Reply:
The other similar components are passing it as a prop. Maybe I can do that too